### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by bcaton85**
A series of edits based on upstream feedback:
- Limited PipelineRun attestations to `oci` and `tekton` backends only
- Renamed `ObjectSigner` receiver to `o`
- Replaced `pkg/chains/formats/intotoite6/util` package with  `attest`, `extract`, and `objectloader` packages
- Renamed variables to correct casing
- Documented `pkg/chains/objects/objects.go`
- Removed `versioned.Interface` parameter from `StorePayload` interface
- Refactored logging in the PipelineRun reconciler
- Moved methods creating Tekton objects for tests into `pkg/internal/tekton`

### CODE REVIEW

File: pkg/chains/formats/intotoite6/attest/attest.go [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 The Tekton Authors
```
Comment (lcarva) [inline pkg/chains/formats/intotoite6/attest/attest.go:2]: Time to move past that year :wink:

File: pkg/chains/formats/intotoite6/objectloader/objectloader.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+package objectloader
+
+import (
+	"encoding/json"
+	"os"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+)
+
+func TaskRunFromFile(f string) (*v1beta1.TaskRun, error) {
```
Comment (lcarva) [inline pkg/chains/formats/intotoite6/objectloader/objectloader.go:26]: This and `PipelineRunFromFile` are only used in tests. Should it go under `internal` as well?

File: pkg/chains/formats/intotoite6/attest/attest.go (hunk 1 above)
Comment (bcaton85) [reply pkg/chains/formats/intotoite6/attest/attest.go:2]: Never understood exactly how those copyright statements worked lol I can update

File: pkg/chains/formats/intotoite6/objectloader/objectloader.go (hunk 2 above)
Comment (bcaton85) [reply pkg/chains/formats/intotoite6/objectloader/objectloader.go:26]: yup

**Review by lcarva — APPROVED** [review]

**Pull request merged**